docs/source/python_api/mlflow.metrics.rst

```diff
@@ -163,8 +163,6 @@ Parameters:
         # with a static dataset and using `extra_metrics`
         mlflow.evaluate(
             data=data,
-            predictions="retrieved_docs",
-            targets="ground_truth_docs",
```

Comment:
wondering why we need this change

Reply:
sorry, I didn't read the PR description carefully. These are duplicates. Makes sense to remove them.